wp-content/themes/theme_name/footer.php

```diff
@@ -1,9 +1,11 @@
-    </main>
-    <footer class="footer">
-      <div class="footer__container">
-        <!-- insert content here -->
-      </div>
-    </footer>
-    <?php wp_footer(); ?>
-  </body>
-</html>
+<?php
+/**
+ * Display footer
+ */
+?>
+
+</main>
+
+<?php include(locate_template('template-parts/footer/footer-content.php')); ?>
```

Comment:
Why not `get_template_part()`? It's a wp built in function to fetch template files.